### ISSUE TITLE
compile error when set renderer to null

### ISSUE THREAD

**Issue by distolma** [electron-userland/electron-webpack#14]
compile throw an error when `render.sourceDirectory` is null

package.json config:
```
"electronWebpack": {
    "whiteListedModules": [
      "chokidar"
    ],
    "main": {
      "sourceDirectory": "src"
    },
    "renderer": {
      "sourceDirectory": null
    }
  }
```

run: 
```
electron-webpack dev
```

error: 
```
Error: Cannot find entry file src\renderer\index.ts (or .js)
    at D:\electron\node_modules\electron-webpack\src\main.ts:290:11
From previous event:
    at computeEntryFile (D:\electron\node_modules\electron-webpack\out\main.js:96:22)
    at D:\electron\node_modules\electron-webpack\src\main.ts:182:35
From previous event:
    at WebpackConfigurator.configure (D:\electron\node_modules\electron-webpack\out\main.js:349:11)
....
```

but when run only `electron-webpack` everything is ok

**Comment by develar** [comment]
To disable renderer code processing, please set `renderer` to `null` :)

**Comment by distolma** [comment]
@develar like this?
```
"electronWebpack": {
    "whiteListedModules": [
      "chokidar"
    ],
    "main": {
      "sourceDirectory": "src"
    },
    "renderer": null
  }
```

if you notised, i`m already set `sourceDirectory` to `null`

**Comment by develar** [comment]
Yes. Setting `sourceDirectory` to `null` is not yet supported (was not expecting).

**Comment by distolma** [comment]
But, `electronWebpack.renderer` accept only object

```
Error: Configuration is invalid:
{
  "renderer": "Should be object"
}
```

**Comment by develar** [comment]
@distolma Please try 1.3.1

**Comment by distolma** [comment]
change config
```
  "electronWebpack": {
    "whiteListedModules": [
      "chokidar"
    ],
    "main": {
      "sourceDirectory": "src"
    },
    "renderer": null
  }
```

but have the same error

```
Error: Cannot find entry file src\renderer\index.ts (or .js)
    at D:\electron\node_modules\electron-webpack\src\main.ts:290:11
From previous event:
```

**Comment by develar** [comment]
Could you please provide full error stack trace?

**Comment by distolma** [comment]
```
Error: Cannot find entry file src\renderer\index.ts (or .js)
    at D:\electron\node_modules\electron-webpack\src\main.ts:290:11
From previous event:
    at computeEntryFile (D:\electron\node_modules\electron-webpack\out\main.js:96:22)
    at D:\electron\node_modules\electron-webpack\src\main.ts:182:35
From previous event:
    at WebpackConfigurator.configure (D:\electron\node_modules\electron-webpack\out\main.js:349:11)
    at D:\electron\node_modules\electron-webpack\src\main.ts:284:48
    at Generator.next (<anonymous>)
    at runCallback (timers.js:781:20)
    at tryOnImmediate (timers.js:743:5)
    at processImmediate [as _immediateCallback] (timers.js:714:5)
From previous event:
    at configure (D:\electron\node_modules\electron-webpack\out\main.js:82:22)
    at Object.getAppConfiguration (D:\electron\node_modules\electron-webpack\src\main.ts:23:55)
    at module.exports.env (D:\electron\node_modules\electron-webpack\webpack.app.config.js:1:47)
    at handleFunction (D:\electron\node_modules\webpack\lib\prepareOptions.js:26:13)
    at prepareOptions (D:\electron\node_modules\webpack\lib\prepareOptions.js:11:13)
    at requireConfig (D:\electron\node_modules\webpack\bin\convert-argv.js:98:14)
    at D:\electron\node_modules\webpack\bin\convert-argv.js:104:17
    at Array.forEach (<anonymous>)
    at module.exports (D:\electron\node_modules\webpack\bin\convert-argv.js:102:15)
    at yargs.parse (D:\electron\node_modules\webpack\bin\webpack.js:171:41)
    at Object.Yargs.self.parse (D:\electron\node_modules\yargs\yargs.js:533:18)
    at Object.<anonymous> (D:\electron\node_modules\webpack\bin\webpack.js:152:7)
    at Module._compile (module.js:573:30)
    at Object.Module._extensions..js (module.js:584:10)
    at Module.load (module.js:507:32)
    at tryModuleLoad (module.js:470:12)
    at Function.Module._load (module.js:462:3)
    at Module.require (module.js:517:17)
    at require (internal/module.js:11:18)
error Command failed with exit code 1.
```

**Comment by develar** [comment]
Workaround for now: `electron-webpack main` (since you need compile only main).

**Comment by distolma** [comment]
thank you, it's works.
one more think, how to change output directory?

**Comment by develar** [comment]
> one more think, how to change output directory?

Ну вот зачем? For what? In this case you will be also forced to change provided electron-builder  config and write your own. 

Custom output dir is not yet supported.

**Comment by develar** [comment]
Fixed in 1.4.0
